### ISSUE TITLE
[date skeleton] CLDR data v31 changes `MMMMW` to `MMMMW-count-one` and `MMMMW-count-other`

### ISSUE THREAD

**Issue by ljqx** [globalizejs/globalize#776]
Now we can use neither `skeleton: "MMMMW"` nor `skeleton: "MMMMW-count-one"`, same for `"yw"`. This makes no way to format a date to `week of {{placeholder}}`.

See 
1. git blame: https://github.com/unicode-cldr/cldr-dates-modern/blame/57630f9b99ee5514b47f59f95db3be4ca0844d5d/main/en/ca-gregorian.json#L368
2. CLDR doc: http://www.unicode.org/reports/tr35/tr35-47/tr35-dates.html#Week_Data

**Comment by rxaviers** [comment]
True, `W` pattern isn't implemented; but I'm open to include it. Would you like to work on it?

**Comment by ljqx** [comment]
@rxaviers my pleasure :)

**Comment by JayChandler** [comment]
Any updates on this ?
